File: Backend/src/main/java/com/example/backend/Entities/Story.java
package com.example.backend.Entities;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import java.util.Date;
import java.util.List;
import java.util.Set;

@Document
@NoArgsConstructor
@AllArgsConstructor
@Data
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Story {


    @Id
    String id;
    String Content ;
    Set<FileEntity>images;
    Integer likes;
    Integer views;
    Date date;
    String storyedBy;
    List<String> tags;
    String artistName;
    String trackName;




}

File: Frontend/vitanova/src/app/FrontOffice/Models/story.ts
import { FileEntity } from "./file-entity";

export class Story {


   id!:String;
   Content!:String;
   images!:FileEntity[] ;
   likes!:number;
   views!:number;
   date!:Date;
   postedBy!:String;
   tags!: string[];



}
